image/base/Dockerfile

```diff
@@ -121,3 +120,8 @@ ADD boot /sbin/boot
 ADD cron /etc/service/cron/run
 ADD rsyslog /etc/service/rsyslog/run
 ADD cron.d_anacron /etc/cron.d/anacron
+
+# Create discourse user and /var/www location for both
+# discourse and dev images.
+RUN useradd discourse -s /bin/bash -m -U &&\
```

Comment:
Fair enough... In that case it may make sense to create an image/layer between base and discourse & discourse_dev.